src/applications/mhv-medical-records/hooks/useAcceleratedData.js

```diff
@@ -72,13 +72,9 @@ const useAcceleratedData = () => {
 
   const isAcceleratingAllergies = useMemo(
     () => {
-      return (
-        isAcceleratedDeliveryEnabled &&
-        isAcceleratingAllergiesEnabled &&
-        isCerner
-      );
+      return isAcceleratedDeliveryEnabled && isAcceleratingAllergiesEnabled;
```

Comment:
SUGGESTION: I like this change, I would look to export the isCerner and use that as the determination if a Veteran is a Cerner patient or not in the container